### ISSUE TITLE
At zoom 3, Somaliland is shown as a country

### ISSUE THREAD

**Issue by SomeoneElseOSM** [gravitystorm/openstreetmap-carto#4803]
### Expected behavior

In accordance with https://wiki.osmfoundation.org/w/images/d/d8/DisputedTerritoriesInformation.pdf , OSM doesn't think that Somaliland is a country.  It's admin_level=3, not admin_level=2.  Somalia is admin_level=2, of course, and includes Somaliland.

### Actual behavior

At zoom 3, Somaliland "looks like a country".  See https://community.openstreetmap.org/t/why-is-somaliland-its-own-country-in-this-map/97568/10 .

### Screenshots with links illustrating the problem

See https://community.openstreetmap.org/t/why-is-somaliland-its-own-country-in-this-map/97568/13 for details.

As that comment says, I suspect that OSM Carto's still using https://naturalearth.s3.amazonaws.com/110m_cultural/ne_110m_admin_0_boundary_lines_land.zip at zooms 1-3, and https://github.com/nvkelso/natural-earth-vector/tree/b70b7b8766c3c83d5f5145cb6f1c93921d383d3f/geojson suggests that that includes the Somaliland / Rest of Somalia border.

Obviously I'm just commenting on the map style here; there's a whole other discussion about what admin level Somaliland ought to be that isn't relevant here (FWIW I think that the OSM data has got it about right at admin_level=3, based on the lack of recognition of Somaliland internationally, despite how autonomous it is).

**Comment by matkoniecz** [comment]
Sounds like consequence of #2172

**Comment by imagico** [comment]
Thanks for reporting this.

This is a symptom of #2172.  Even if in principle this could be solved without solving #2172 it is doubtful we want to do that.  Solving #2172 would quite definitely solve this.

What is needed to solve #2172 is not rocket science - it is roughly sketched in https://github.com/gravitystorm/openstreetmap-carto/issues/2172#issuecomment-604945615.  But it is not trivial either to do so in a robust and flexible fashion - not something you can quickly hack together by combining a few standard geometry library functions in your programming language of choice.  My assessment is that most OSM-Carto developers who would be able to do that do not have the capacity to do so in their limited volunteer time.  Considering a robust open source solution for this would be of use for many other map styles as well it would be an excellent investment in the OSM community's map design abilities for any organization interested in supporting OSM to finance a project for this.

**Comment by ZeLonewolf** [comment]
I concur that this issue results from using Natural Earth boundaries. The tile server that is presently powering Americana uses purely OSM-derived boundaries (it's an option I enabled in our tile server), and it displays the [correct behavior](https://zelonewolf.github.io/openstreetmap-americana/#map=3/29.43/37.02) as Andy describes for `admin_level=3` boundaries. You'll note that Somaliland appears at the same time as England/Wales or England/Scotland boundaries.

**Comment by matkoniecz** [comment]
Looking into it: z1, z2 are also affected, not only z3

**Comment by imagico** [comment]
Yes, Natural Earth boundaries are used at z1-z3 and this is the only remaining non-OSM geodata we use in the map at this time.

There is the standing suggestion in https://github.com/gravitystorm/openstreetmap-carto/issues/2172#issuecomment-1500900302 to simply remove these until a proper solution is developed - but it is unclear so far if this has consensus support from the maintainers.

**Comment by Hurbad** [comment]
Hi @imagico, do you know if there has been any recent changes on this? Seems like the bug is still there 

![Image](https://github.com/user-attachments/assets/131c2e22-a9ed-4188-94a7-ae811c097499)
![Image](https://github.com/user-attachments/assets/84e50380-6605-43f8-a338-3eb3b3668fcd)

ein Wink mit dem Zaunpfahl 😉 Definitely a good project to sponsor to fix

**Comment by matkoniecz** [comment]
> Hi @imagico, do you know if there has been any recent changes on this? Seems like the bug is still there

1) nothing was linked in this PR so, apparently no

2) where you see this? that does not look like OSM Carto

**Comment by Hurbad** [comment]
Hi @imagico and @matkoniecz , do you know if there's been any updates on this and if not, is there a way to sponsor this improvement.

**Comment by Hurbad** [comment]
Of course, I'm not suggesting that a viable solution could be hacked together as it would probably just break another thing.

I meant sponsoring the removal of the z1-3 admin boundaries as it is messing with other map services based on it (especially ones shown in news and tech demos). I think our company found a way around it or someone else did. 

Do you happen to know any developer who would be interested in removing the boundaries as a temporary fix whilst we wait for a better process?

**Comment by imagico** [comment]
I see.  Removing the NE boundary display at z1-3 is not a big change and as you can see someone already submitted a change for that now.

There is some chance that this change might make the need for developing a proper solution more broadly visible.
